snapcraft/storeapi/__init__.py

```diff
@@ -182,6 +189,7 @@ def _refresh_if_necessary(self, func, *args, **kwargs):
         try:
             return func(*args, **kwargs)
         except errors.StoreMacaroonNeedsRefreshError:
+            print('REFRESHING')
```

Comment:
Is this an error message, or progress text?

Reply:
Whoops, a debug message I missed, thank you :grin: .